build.gradle

```diff
@@ -63,8 +64,16 @@ dependencies {
 
   signature 'org.codehaus.mojo.signature:java17:1.0@signature'
 
-  testCompile 'org.spockframework:spock-core:0.7-groovy-1.8'
+  testFixturesImplementation 'org.jenkins-ci.main:jenkins-test-harness:2.44'
+
   jenkinsTest 'org.jenkins-ci.main:jenkins-test-harness:2.44@jar'
+
+  testCompile 'org.spockframework:spock-core:0.7-groovy-1.8'
+
+  jenkinsNextTestCompile "org.jenkins-ci.main:jenkins-war:${nextJenkinsVersion}"
+  jenkinsNextTestCompile 'org.spockframework:spock-core:1.2-groovy-2.4', { exclude module: 'groovy-all' }
+  jenkinsNextTestCompile 'io.jenkins:configuration-as-code:1.4@jar'
```

Comment:
Have you seen pull request #65?

Reply:
Yes, I did. I wanted to change that separately.